fontforgeexe/fontinfo.c

```diff
@@ -8433,16 +8447,6 @@ return;
     vgcd[13].gd.u.list = ibmfamily;
     vgcd[13].creator = GListButtonCreate;
 
```

Comment:
Because the concerned checkbox is at the end of the window and its positioning is relative to that of the element I added (statically so).